src/main/java/vazkii/botania/common/lib/ModTags.java

```diff
@@ -184,10 +184,10 @@
 		public static final Tag.Identified<Block> LIVINGROCK = tag("livingrock");
 		public static final Tag.Identified<Block> LIVINGWOOD = tag("livingwood");
 
-		public static final Tag.Identified<Block> BLOCKS_ELEMENTIUM = forgeTag("storage_blocks/elementium");
-		public static final Tag.Identified<Block> BLOCKS_MANASTEEL = forgeTag("storage_blocks/manasteel");
-		public static final Tag.Identified<Block> BLOCKS_QUARTZ = forgeTag("storage_blocks/quartz");
-		public static final Tag.Identified<Block> BLOCKS_TERRASTEEL = forgeTag("storage_blocks/terrasteel");
+		public static final Tag.Identified<Block> BLOCKS_ELEMENTIUM = tag("elementium_blocks");
+		public static final Tag.Identified<Block> BLOCKS_MANASTEEL = tag("manasteel_blocks");
+		public static final Tag.Identified<Block> BLOCKS_QUARTZ = commonTag("quartz_blocks");
+		public static final Tag.Identified<Block> BLOCKS_TERRASTEEL = tag("terrasteel_blocks");
```

Comment:
so is it commonTag or tag? You're mixing both here.

Reply:
there's a common tag for quartz, mods that add their own should have it in their own namespace